Source/BusinessMomentsAdaptorImage/server.ts

```diff
@@ -1,36 +1,45 @@
 import express, { Application, Request, Response } from 'express';
 import bodyParser from 'body-parser';
 import process from 'process';
-
+import * as dataController from './dataController';
+import rawData from './dataschema';
 
 export function createServer() {
     const app: Application = express();
 
-    // Tell express to use body-parser's JSON parsing
+    // Tell express to use body-parser's JSON parsing.
     app.use(bodyParser.json());
 
     app.post('/api/webhooks-ingestor', (req: Request, res: Response) => {
         if (req.headers.authorization === process.env.WH_AUTHORIZATION) {
             console.log(req.body);
+
+            let data = new rawData(req.body);
+            try {
+                data.save();
+            } catch (error) {
+                res.send(error);
```

Comment:
Let's not return the error in the response. If we get an exception we can log it and return the HTTP status code for the internal server error.

We don't know what the error msg might unveil.